discretenet/problems/water_pipe_enhancement/water_pipe_enhancement.py

```diff
@@ -0,0 +1,252 @@
+from pathlib import Path
+import numpy.random as random
```

Comment:
ditto with above about numpy random import and import order

Reply:
Updated